### ISSUE TITLE
15px wide borders on eSRRF reconstructions

### ISSUE THREAD

**Issue by arthursw** [HenriquesLab/NanoPyx#146]
Hi,

I noticed 15px wide borders on the eSRRF reconstruction:

<img width="278" height="217" alt="Image" src="https://github.com/user-attachments/assets/5e0bcce0-e491-44fe-b6de-78903621bba4" />

(note: those are images opened in Fiji but computed with NanoPyx ;) )

I used to work with big images so I didn't see that before.

Is there an option to avoid those borders?

**Comment by arthursw** [comment]
Thanks for replying so quickly! 🥳 

How did you do it in NanoJ? Did you do something like [reflection padding](https://docs.pytorch.org/docs/stable/generated/torch.nn.ReflectionPad2d.html#reflectionpad2d)? 

We could do it ourself before the eSRRF processing?

**Comment by arthursw** [comment]
Perfect, thank you very much : )
